censys-subdomain-finder.py

```diff
@@ -10,95 +14,131 @@
 
 load_dotenv()
 
-NON_COMMERCIAL_API_LIMIT = 1000
+USER_AGENT = f"{CensysCerts.DEFAULT_USER_AGENT} (censys-subdomain-finder; +https://github.com/christophetd/censys-subdomain-finder)"
+
 
 # Finds subdomains of a domain using Censys API
 def find_subdomains(domain, api_id, api_secret, limit_results):
     try:
-        censys_certificates = CensysCertificates(api_id=api_id, api_secret=api_secret)
-        certificate_query = 'parsed.names: %s' % domain
+        censys_certificates = CensysCerts(
+            api_id=api_id, api_secret=api_secret, user_agent=USER_AGENT
+        )
+        certificate_query = "names: %s" % domain
         if limit_results:
-            certificates_search_results = censys_certificates.search(certificate_query, fields=['parsed.names'], max_records=NON_COMMERCIAL_API_LIMIT)
+            certificates_search_results = censys_certificates.search(
+                certificate_query, fields=["names"], per_page=100, pages=10
```

Comment:
why 100 and 10? this should be configurable in case you have a commercial API key and can use more

Reply:
Absolutely, so `100` is the max per page size and I figured `10` pages would be the same as `100`*`10`.